aulas/05.md

```diff
@@ -225,9 +226,9 @@ tests/test_app.py::test_root_deve_retornar_200_e_ola_mundo PASSED
 tests/test_app.py::test_create_user PASSED
 ```
 
-## Modificando o endpoint de atualização de usuários
+### Modificando o endpoint de atualização de usuários
```

Comment:
Same as the problem I pointed out above. I just don't know if you did it by design or because it's a typo.